spec/spec_helper.rb

```diff
@@ -0,0 +1,66 @@
+SRC_PATH = File.expand_path("../../src", __FILE__)
+DATA_PATH = File.join(File.expand_path(File.dirname(__FILE__)), "data")
```

Comment:
separation of code and data...and also if update-alternatives output changes, then it is much easier to update test code to reflect reality